Makefile

```
@@ -129,9 +129,8 @@ devtest:
 .PHONY: sanitize-pr
 sanitize-pr:
 	./hack/bin/generate-local-nix-packages.sh
-	make formatf
 	make hlint-inplace-pr
-	make hlint-check-pr  # sometimes inplace has been observed not to do its job very well.
+	make format
```

```suggestion
	make format
	make formatc  # (at some point in the past, this has caught ormolu screwing up with inplace)
```

this *might* be fixed now with the `--check-inplace` argument, but can we be certain?

but if you prefer, we can remove this and see if it happens again.

---

`--check-idempotence`? i have removed `formatc`, i think...

---

i was hoping that ormolu by now is idempotent or at least the idempotency check works reliably :eyes: